leshan-demo-server/src/main/java/org/eclipse/leshan/demo/server/servlet/EventServlet.java

```diff
@@ -303,6 +307,22 @@ public EventServlet(LeshanServer server) {
         this.mapper = objectMapper;
     }
 
+    private String formatTimestampedNodes(TimestampedLwM2mNodes data) {
```

Comment:
The current `TimestampedLwM2mNodes.toString()` is not good enough ? 
Should we improve it instead of adding that new method  ?